packages/aws-cdk-lib/aws-eks/lib/cluster.ts

```diff
@@ -732,6 +732,15 @@ export interface ClusterOptions extends CommonClusterOptions {
    * @default - Resources will be deleted.
    */
   readonly removalPolicy?: RemovalPolicy;
+
+  /**
+   * The current deletion protection setting for the cluster.
+   * When true, deletion protection is enabled and the cluster cannot be deleted until protection is disabled.
+   * When false, the cluster can be deleted normally. This setting only applies to clusters in an active state.
+   *
+   * @default - none
```

Comment:
Could you please describe the default value or  behavior that happens if this property is not supplied?
```suggestion
   // default value
   * @default false
   // default behaviour
   * @default - deletion protection is disabled
```